integration-tests/smoke/vrfv2_test.go

```diff
@@ -977,14 +962,22 @@ func TestVRFV2WithBHS(t *testing.T) {
 		_, err = vrfContracts.BHS.GetBlockHash(testcontext.Get(t), big.NewInt(int64(randRequestBlockNumber)))
 		require.Error(t, err, "error not occurred when getting blockhash for a blocknumber which was not stored in BHS contract")
 
+		sethClient, err := testEnv.GetSethClient(chainID)
+		require.NoError(t, err, "Getting Seth client shouldn't fail")
+
 		var wg sync.WaitGroup
 		wg.Add(1)
-		_, err = actions.WaitForBlockNumberToBe(randRequestBlockNumber+uint64(*configCopy.VRFv2.General.BHSJobWaitBlocks), evmClient, &wg, time.Minute*1, t)
+		_, err = actions.WaitForBlockNumberToBe(
+			randRequestBlockNumber+uint64(*configCopy.VRFv2.General.BHSJobWaitBlocks),
+			sethClient,
+			&wg,
+			time.Minute*1,
+			t,
+			l,
+		)
```

Comment:
why even use wait group if it only has 1 goroutine?

Reply:
@Tofel I have no idea :) This goroutine was there, I just updated the code to work with Seth. But, I can remove it